docker-cloud.yml

```diff
@@ -2,4 +2,4 @@ web:
   name: "loklak"
   image: mariobehling/loklak
   ports:
-    - "80"
+    - "443"
```

Comment:
Please don't delete the `80` instead add 2 port values, that's what was expected
```
- "80"
- "443"
```